requirements.txt

```diff
@@ -1,5 +1,3 @@
 plost
 streamlit
-pandas
-numpy
-plotly
+plotly
```

Comment:
numpy and pandas are installed by streamlit, so I removed them to it's not redundant. The original main version did not have these, so let me know if there was another reason to add them back.

Reply:
Thanks for catching this. I would rather we remove any redundancy.